tests/clv/models/test_shifted_beta_geo.py

```diff
@@ -76,71 +121,61 @@ def test_model(self, alpha_prior, beta_prior):
             "customer_id": tuple(range(self.N)),
         }
 
-    def test_invalid_t_churn(self):
+    def test_invalid_t_churn(self, default_model_config):
         match_msg = "t_churn must respect 0 < t_churn <= T"
-        customer_id = range(3)
+        dataset = {
+            "customer_id": range(3),
+            "t_churn": [10, 10, np.nan],
+            "T": 10,
+        }
 
+        dataset["t_churn"] = [10, 10, np.nan]
         with pytest.raises(ValueError, match=match_msg):
             ShiftedBetaGeoModelIndividual(
-                customer_id=customer_id,
-                t_churn=[10, 10, np.nan],
-                T=10,
+                data=pd.DataFrame(dataset), model_config=default_model_config
             )
-
+        dataset["t_churn"] = [10, 10, 11]
         with pytest.raises(ValueError, match=match_msg):
             ShiftedBetaGeoModelIndividual(
-                customer_id=customer_id,
-                t_churn=[10, 10, 11],
-                T=10,
+                data=pd.DataFrame(dataset), model_config=default_model_config
             )
-
+        dataset["t_churn"] = [-1, 8, 9]
+        dataset["T"] = [8, 9, 10]
         with pytest.raises(ValueError, match=match_msg):
             ShiftedBetaGeoModelIndividual(
-                customer_id=customer_id,
-                t_churn=[-1, 8, 9],
-                T=[8, 9, 10],
+                data=pd.DataFrame(dataset),
             )
 
-    def test_model_repr(self):
```

Comment:
it wasn't removed, as I explained with the __repr__ in basic.py, I was taught the convention of putting __functions on top of the file, so I feel strange having functions testing exactly that in other place, it's still in the file, but here marked as 'added' on the top of the file

Reply:
Stylistic conventions should be enforced by pre-commit hooks like black, otherwise every developer will have their own opinion and change the same code around without need. This makes git changes harder to review both in this PR and for future developers.

Reply:
I won't change anything style-wise from now on here, promise!